cli/consumer_command.go

```diff
@@ -163,6 +163,7 @@ func configureConsumerCommand(app commandHost) {
 	consInfo.Arg("stream", "Stream name").StringVar(&c.stream)
 	consInfo.Arg("consumer", "Consumer name").StringVar(&c.consumer)
 	consInfo.Flag("json", "Produce JSON output").Short('j').UnNegatableBoolVar(&c.json)
+	consInfo.Flag("force", "Force info without prompting").Short('f').UnNegatableBoolVar(&c.force)
```

Comment:
I think force is maybe the wrong flag for this - I have been considering a general higher level flag that would apply equally to streams, consumers and a few other places that disables the lists since we have this annoyance a few places.

So a top level flag for --no-select maybe that then applies to all the various places.  That would need to feed into a new option setting.

I can take care of that if its too much for you to tackle?

Reply:
yes of course. our idea is to use nats cli for some back scripts.

Reply:
OK will see about that later this week

Reply:
I think though if thsoe scripts run without a TTY it wont prompt

Reply:
Lets just change these 2 force flags to:

```
consInfo.Flag("select", "Select consumers from a list").Default("true").UnNegatableBoolVar(&c.force)
```

this will be a bit less confusing than force - but then we just reuse the force boolean, then its good to go I think.

So you'd just pass --no-select